### DISCUSSION TITLE
Pausing the carousel

### DISCUSSION THREAD

**Discussion by Manpreet-Bhatti** [vadymshymko/react-simply-carousel#196] [Q&A]
Would it be possible to pause the carousel `onHover` without it pausing once the specified amount of slides to show is shown? For example, if I want it to show 5 slides and I've created a mechanism to pause `onHover` (using `onMouseEnter`), it'll keep scrolling until 5 slides are visible instead of hard-stopping. Note that this is for when the `infinite` and `centerMode` is enabled.

**Comment by vadymshymko** [comment]
Hi @Manpreet-Bhatti 
Interesting point.  I'll think about it and come back with more details later.
